packages/non-core/coffeescript/package.js

```diff
@@ -6,12 +6,12 @@ Package.describe({
   // so bumping the version of this package will be how they get newer versions
   // of `coffeescript-compiler`. If you change this, make sure to also update
   // ../coffeescript-compiler/package.js to match.
-  version: '2.0.3_2'
+  version: '2.0.3-2-beta161.13'
 });
 
 Package.registerBuildPlugin({
   name: 'compile-coffeescript',
-  use: ['caching-compiler@1.1.9', 'ecmascript@0.8.3', 'coffeescript-compiler@=2.0.3_2'],
+  use: ['caching-compiler@1.1.9', 'ecmascript@0.8.3', 'coffeescript-compiler@~2.0.3-2-beta161.13'],
```

Comment:
This version constraint can go back to being exact (`@=`) once we remove the `-beta161.n` suffix.